File: 백준/Bronze/2588. 곱셈/곱셈.py
a = int(input())
b = int(input())

x = int(b / 100);
z = int(b % 10);
y = int((b - (x * 100 + z)) / 10)

print(a * z)
print(a * y)
print(a * x)
print(a * b)

File: 백준/Bronze/2588. 곱셈/곱셈.js
var fs = require('fs');
var input = fs.readFileSync('/dev/stdin').toString().trim().split('\n');
var a = parseInt(input[0]);
var b = parseInt(input[1]);
console.log(
  a * (b % 10) +
    '\n' +
    a * (Math.floor(b / 10) % 10) +
    '\n' +
    a * Math.floor(b / 100) +
    '\n' +
    a * b
);